src/develop/ui/patterns/web/navigation/wizard.md

```diff
@@ -3,153 +3,181 @@ tags: runtime-traditionalweb;
+1. Repeat steps 13 and 14 for the **Next** button. Enter `CurrentStep < 4` as the **Condition** property. This ensures the **Next** button is only displayed when applicable, and the user can never go above the number of steps in the wizard.
 
-    ![](images/wizard-image-5.png)
+**Add content to the wizard**
 
-1. Drag an AjaxRefresh to refresh your content container on the screen.
+1. From the Toolbox, drag the If widget onto the main content area of your application's screen, just below the Wizard Pattern.
 
-1. Set the OnClick of the Continue Button to the action WizardNavigation with Ajax Submit as the Method. Set the IsNext parameter to True.
+    ![](images/wizard-image-30.png)
+
+1. On the **Properties** tab, in the **Condition** property, enter the following:
+
+   `CurrentStep = 1`
+           
+    ![](images/wizard-image-19.png)
+
+    You have now created a condition that controls what is displayed on screen when the step is acive.
+
+1. Repeat steps 1 and 2 for all of the Wizard Items. Replace **Condition** property with 2, 3, and 4 respectively.
+
+1. In the top section (True) of If widgets, create the relevant content for each of the steps.
+
+     The following example shows a Shipping Details step that contains labels and input boxes.
+
+      ![](images/wizard-image-18.png)
+
+**Create the wizard navigation**
+
+To ensure that all of the information the user enters is passed from step to step, you must create a screen action.
+
+
+
+) for the **Previous** and **Next** navigation buttons.
+
+1. Select the **Next** button.
+
+1. On the **Properties** tab, from the **Destination** drop-down, select **New Screen Action**.
+
+    ![](images/wizard-image-22.png)
+
+1. From the Toolbox, drag the Assign block onto your screen and set the **Variable** property to ``CurrentStep`` and the Value property to ``CurrentStep + 1``. This ensures that user inputs get passed from step to step.
+
+    ![](images/wizard-image-21.png)
+
+1. Repeat steps 2 and 3 for the **Previous** button.
+
+After following all of the steps in each of the sections, you can publish the module, and test the Wizard in your app.
+
+## Properties
+
+### Wizard
+
+| **Property** |  **Description** | 
+|---|---|
+| Orientation (Orientation Identifier): Optional  |  Set the wizard orientation. By default the Wizard displays horizontally. The Orientation drop-down shows the different orientation options: <ul><li> Horizontal (_Entities.Orientation.Horizontal_) - The wizard displays horizontally </li></ul> <ul><li>Vertical (_Entities.Orientation.Vertical_) - The wizard displays vertically</ul></li>|
+| ExtendedClass (Text): Optional  |  Add custom style classes to the block. |
+
+### Wizard Item
+
+| **Property** |  **Description** | 
+|---|---|
+| Step (Step Identifier): Mandatory | Set the step. Using an expression, you can set the status of a step to being active (the step the user is on) or inactive (a step the user has already completed or has yet to complete).| 
```

Comment:
"Set the Step" doesn't mean much. :) Maybe something like:
Set the status of the Wizard Item relative to the current step of the Wizard.
Also, we need the actual values: Active, Past and Next, to explain what each value means.